spec/lib/myfinance/resources/person_spec.rb

```diff
@@ -0,0 +1,226 @@
+require "spec_helper"
+
+describe Myfinance::Resources::Person do
+  describe "#find_all", vcr: true do
+    context "when success" do
+      subject { client.people.find_all }
+
+      it "returns a find_all of people" do
+        expect(subject.class).to eq(Myfinance::Entities::PersonCollection)
+        expect(subject.collection.first.class).to eq(Myfinance::Entities::Person)
+        expect(subject.collection.length).to eq(1)
+        expect(subject.collection.first.id).to eq(199424)
+        expect(subject.collection.first.account_id).to eq(3613)
+        expect(subject.collection.first.address).to eq("")
+        expect(subject.collection.first.address_number).to eq("")
+        expect(subject.collection.first.city).to eq("")
+        expect(subject.collection.first.complement).to eq("")
+        expect(subject.collection.first.country).to eq("")
+        expect(subject.collection.first.created_at).to eq(DateTime.parse("2015-11-12T14:34:37-02:00"))
```

Comment:
Do not use `DateTime.parse` without zone. Use one of `Time.zone.parse`, `DateTime.parse.current`, `DateTime.parse.in_time_zone`, `DateTime.parse.utc`, `DateTime.parse.getlocal`, `DateTime.parse.iso8601`, `DateTime.parse.jisx0301`, `DateTime.parse.rfc3339`, `DateTime.parse.to_i`, `DateTime.parse.to_f` instead.